idx/cassandra/cassandra_test.go

```diff
@@ -285,3 +387,46 @@ func BenchmarkLoad(b *testing.B) {
 	ix.Init()
 	ix.Stop()
 }
+
+func BenchmarkIndexingWithUpdates(b *testing.B) {
```

Comment:
right but it's not updating anything in cassandra. and since this test is called `WithUpdates` and it's in idx/cassandra/cassandra_test.go this leads to believe it benchmarks updates in cassandra. so maybe just call it `BenchmarkIndexingUpdatesMemoryNotCassandra` or something